recipes/shyaml/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "shyaml" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80650ebfe6fa2e16083451d515207472d60990c1c15fc0fd607c27077790ac23
+  patches:
+    # this is needed to make conda entry point work.
+    - cli-args.patch
+
+build:
+  noarch: python
+  number: 0
+  script: |
+    $PYTHON setup.py install
```

Comment:
Since we are using `setuptools`, do we need the other arguments here? Was there a reason `pip` didn't work for this?

Reply:
Yeah pip fails to install for some reason, might be because it is using `d2to1` and other nonstandard hacks